openslides/core/config_variables.py

```diff
@@ -270,3 +271,13 @@ def get_config_variables():
         weight=311,
         group='Logo',
         hidden=True)
+
+    yield ConfigVariable(
+        name='logo_web_header',
+        default_value={
+            'display_name': 'Web interface header logo',
+            'path': ''},
+        input_type='logo',
+        weight=311,
```

Comment:
Please use new weight number. (I know that this is hidden but this may change in future. :-)